app/config.py

```diff
@@ -29,6 +29,7 @@ class QueueNames(object):
     CALLBACKS = 'service-callbacks'
     LETTERS = 'letter-tasks'
     ANTIVIRUS = 'antivirus-tasks'
+    TEMPLATE_PREVIEW = 'template-preview-tasks'
```

Comment:
Not a big deal - but should this be called `validate-letter-tasks`

Reply:
I went for `template-preview-tasks` to match `antivirus-tasks` above it, but I'll change it